lib/slack_transformer/html/paragraph.rb

```diff
@@ -0,0 +1,38 @@
+require 'nokogiri'
+
+module SlackTransformer
+  class Html
+    class Paragraph
+      attr_reader :input
+
+      def initialize(input)
+        @input = input
+      end
+
+      def to_slack
+        fragment = Nokogiri::HTML.fragment(input)
+        previous = nil
+        
+        fragment.children.each do |child|
+          if child.name == 'p' 
+            
+            if previous.nil?
+              child.replace("#{child.children.to_html}")
+            else
+              child.replace("\n#{child.children.to_html}")
+            end
+
+            previous = child.children.empty? ? nil : 'p'
```

Comment:
i'm not sure i'm fully following this line :o can you explain it a bit to me?

Reply:
Oh looking at the tests, i see what you mean. Maybe a quick comment here to denote that this is a special edge case where the p tag is empty

Reply:
Done,